pymilvus/client/async_grpc_handler.py

```diff
@@ -0,0 +1,2449 @@
+import base64
+import copy
+import json
+import socket
+import time
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Mapping, Optional, Union
+from urllib import parse
+
+import grpc
+from grpc._cython import cygrpc
+
+from pymilvus.decorators import ignore_unimplemented, retry_on_rpc_failure, upgrade_reminder
+from pymilvus.exceptions import (
+    AmbiguousIndexName,
+    DescribeCollectionException,
+    ErrorCode,
+    ExceptionsMessage,
+    MilvusException,
+    ParamError,
+)
+from pymilvus.grpc_gen import common_pb2, milvus_pb2_grpc
+from pymilvus.grpc_gen import milvus_pb2 as milvus_types
+from pymilvus.settings import Config
+
+from . import entity_helper, interceptor, ts_utils, utils
+from .abstract import AnnSearchRequest, BaseRanker, CollectionSchema, MutationResult, SearchResult
+from .async_interceptor import async_header_adder_interceptor
+from .asynch import (
+    CreateIndexFuture,
+    FlushFuture,
+    LoadPartitionsFuture,
+    MutationFuture,
+    SearchFuture,
+)
+from .check import (
+    check_pass_param,
+    is_legal_host,
+    is_legal_port,
+)
+from .constants import ITERATOR_SESSION_TS_FIELD
+from .prepare import Prepare
+from .types import (
+    BulkInsertState,
+    CompactionPlans,
+    CompactionState,
+    DatabaseInfo,
+    DataType,
+    ExtraList,
+    GrantInfo,
+    Group,
+    IndexState,
+    LoadState,
+    Plan,
+    PrivilegeGroupInfo,
+    Replica,
+    ResourceGroupConfig,
+    ResourceGroupInfo,
+    RoleInfo,
+    Shard,
+    State,
+    Status,
+    UserInfo,
+    get_cost_extra,
+)
+from .utils import (
+    check_invalid_binary_vector,
+    check_status,
+    get_server_type,
+    is_successful,
+    len_of,
+)
+
+
+class AsyncGrpcHandler:
```

Comment:
Please inherit GrpcHandler class, so that most sync methods won't need to re-write, only need to add async methods here.

Reply:
Since AsyncGrpcHandler holds async methods only, no need to inherit